tests/src/python/test_qgssymbollayer.py

```diff
@@ -89,7 +89,7 @@
 
 TEST_DATA_DIR = unitTestDataPath()
 
-if StrictVersion(PYQT_VERSION_STR) < StrictVersion('5.7'):
+if Version(PYQT_VERSION_STR) < Version('5.7'):
```

Comment:
Same here -- this can be dropped entirely

Reply:
done